node.js

```diff
@@ -8,30 +16,43 @@ const { Worker } = require("worker_threads");
 const PROTO_PATH = path.resolve(__dirname, "./protos/chord.proto");
 
 const packageDefinition = protoLoader.loadSync(PROTO_PATH, {
-  keepCase: true,
-  longs: String,
-  enums: String,
-  defaults: true,
-  oneofs: true
+    keepCase: true,
+    longs: String,
+    enums: String,
+    defaults: true,
+    oneofs: true
 });
 
 const chord = grpc.loadPackageDefinition(packageDefinition).chord;
 
 const caller = require("grpc-caller");
 
 const HASH_BIT_LENGTH = 3;
+const CHECK_NODE_TIMEOUT_ms = 1000;
+
+const NULL_NODE = { id: null, ip: null, port: null };
 
-const FingerTable = [
-  {
-    start: null,
-    successor: { id: null, ip: null, port: null }
-  }
+let FingerTable = [
```

Comment:
Can you change this to `fingerTable`? Capitalized camel case is usually reserved for classes.